### ISSUE TITLE
Can I host this website locally? File size exceeds maximum.

### ISSUE THREAD

**Issue by gaoyaoxin** [michaelcukier/pytoexe#2]
I want to add a data file but an error occurred. Error: total file(s) size exceed 10MB.
So, is it possible to host this site locally to save time and pressure on distance host?

**Comment by michaelcukier** [comment]
yeah I'll write the instructions to set it up locally in a few days

**Comment by michaelcukier** [comment]
I'm working on it

**Comment by gaoyaoxin** [comment]
thx! The site could not load well sometimes as I tested yesterday and this would help a lot!
